python/cuml/cuml/tests/conftest.py

```diff
@@ -287,8 +287,11 @@ def _get_gpu_memory():
         if len(tokens) > 1:
             gpus_memory.append(int(tokens[0]))
     gpus_memory.sort()
-    max_gpu_memory = ceil(gpus_memory[-1] / 1024)
-    return max_gpu_memory
+    if len(gpus_memory) > 0:
+        max_gpu_memory = ceil(gpus_memory[-1] / 1024)
+        return max_gpu_memory
+    else:
+        return None
```

Comment:
I don't think returning `None` here is a good idea, because it would lead to TypeErrors in many of our (stress) tests.

Reply:
What do you recommend? Would returning 0 be a good solution?